.semaphore/semaphore.yml

```
@@ -62,12 +62,12 @@ blocks:
         commands:
           - docker pull "${IMAGE_NAME}"
           - cp docker-compose.linux.yml docker-compose.override.yml
+          - docker-compose run app bin/rails db:create db:schema:load
```

Maybe we use `db:setup`?
```suggestion
          - docker-compose run app bin/rails db:setup
```

---

We tried it before but this command sometimes failed on CI without the reason. That's why we use `db:create db:schema:load`